src/ms.tex

```diff
@@ -301,18 +301,59 @@ \subsection{Roadmap}
 
 \section{Learn Astropy} \label{sec:learn}
 
-\secauthor{Lía Corrales, David Shupe + Learn team}
+\secauthor{Lía Corrales, David Shupe, Kelle Cruz + Learn team}
+
+\subsubsection{Current status and scope}
+
+{\it Learn Astropy} is an umbrella term that acknowledges the broad educational efforts made by the Astropy Project, which are led by the Learn Team.
+The efforts focus on developing online content and workshops covering astro-specific coding tasks in Python.
+As described in \textsl{[Astropy-ii paper]}, There are four different types of Learn Astropy content: \textit{tutorials}, consisting of Jupyter Notebook lessons that are published in HTML format online; \textit{guides}, which are a series of lessons providing a foundational resource for performing a certain type of astronomical analyses; \textit{examples}, which are snippets of code that showcase a short task that can be performed with Astropy or an affiliated package; and \textit{documentation}, which is contained within the code base.
+This categorization drives content development, infrastructure choices, and the appearance of the Learn Astropy website.
+The Learn Team meets weekly to work on creating, expanding, improving these educational resources.
 
 \begin{itemize}
-\item Current status and scope
-\item Vision for the future (review what we said in v2.0 paper)
-\item User forums and engagement with user base
-\item Workshops
+
+\item {\bf Learn Astropy website:} The website has a new search interface powered by
+Algolia which is used by major e-commerce websites. This work has been
+supported in part by the Dunlap grant.
```

Comment:
How about something like this?

```suggestion
\item {\bf Learn Astropy website:} The Learn Team re-launched the website in 2021 with a new infrastructure platform, built around full-text search and interactive filtering functionality, with the goal of making content discoverable as the Learn Astropy content catalog expands.
This work has been
supported in part by the Dunlap grant.
We have adopted Algolia, a search-as-a-service cloud platform, to store the full-text and metadata records of Learn Astropy's content.
The new Learn Astropy website is a JavaScript (Gatsby/React) application that uses the Algolia service to power its search and filtering user interface.
Our Python-based application, Learn Astropy Librarian, populates data into the Algolia service. We tuned the Librarian around specific content formats (such as Jupyter Notebook-based tutorial pages and Jupyter Book-based guides) to more accurately index content and heuristically extract metadata.
A consequence of the new platform is that we now maintain and compile content separately from the website application itself, enabling new content types.
Tutorials, which as Jupyter Notebook-based, are now compiled into their own Learn Astropy sub-site using nbcollection.
Guides, which are Jupyter Book-based, are also deployed as separate websites using GitHub pages. This architecture opens future possibilities of indexing third-party content, hosted elsewhere, such as on institutional websites.
```

Reply:
I think this is good and I'll commit the suggestion. We probably want to describe "Dunlap grant".